Installer/update-vs.ps1

```diff
@@ -0,0 +1,99 @@
+$installer_name = 'git-credential-manager.exe'
+$title = 'Git Credential Manager for Windows upater for Visual Studio v1.0'
+
+function check_access {
+  # Ensure that we're running elevated, and if we're not then restart as elevated.
+  $current_identity = [System.Security.Principal.WindowsIdentity]::GetCurrent()
+  $current_principle = new-object System.Security.Principal.WindowsPrincipal($current_identity)
+  $administrator_role = [System.Security.Principal.WindowsBuiltInRole]::Administrator
+  
+  if (-not ($current_principle.IsInRole($administrator_role))) {
+    write-host -ForegroundColor Red 'fatal: Access Denied'
+    write-host -ForegroundColor Yellow 'Administrator privilages are required, please run again with administrator privilages.'
+    
+    exit -2
+  }
+}
+
+$current_directory = get-location
+$script_name = $MyInvocation.MyCommand.Name
```

Comment:
$script_name and $script_path appear unused.

Reply:
ah yeah they may be, I'd originally had the script self elevating, but that made automation difficult - same reason for the `ReadKey` stuff - wanted a way to make it possible to read and not just a scary window flash.